pkgs/top-level/all-packages.nix

```diff
@@ -15891,6 +15893,15 @@ with pkgs;
 
   opera = callPackage ../applications/networking/browsers/opera {};
 
+  osm2xmap = callPackage ../applications/misc/osm2xmap {
+    libyamlcpp = pkgs.libyamlcpp.overrideAttrs (oldAttrs: rec {
+      src = pkgs.fetchurl {
+        url = "https://github.com/jbeder/yaml-cpp/archive/release-0.3.0.tar.gz";
+        sha256 = "12aszqw6svwlnb6nzhsbqhz3c7vnd5ahd0k6xlj05w8lm83hx3db";
+      };
+    });
```

Comment:
I would rather add a dedicated `libyamlcpp_0_3` instead so everybody can use it.